buildchain/buildchain/iso.py

```diff
@@ -107,7 +107,53 @@
         destination_directory=constants.ISO_ROOT,
         source_prefix=Path('scripts'),
         task_dep=['_iso_mkdir_root']
-    )
+    ),
+    helper.FileTree(
```

Comment:
I wonder whether it wouldn't be better to turn `metalk8s_cli` into a proper Python package, `sdist` it during build into `_build/...` (not `_build/root`), then 'install' it in `_build/root/whatever` (also during build).

Reply:
I'll give it a try :)

Reply:
OK, I think it's worth a dedicated issue, so we can think about different technical solutions (and what we want to do in the future for this CLI). I'll open one